weldx/geometry.py

```diff
@@ -2023,6 +2025,31 @@ def __init__(self, profile, trace):
         self._profile = profile
         self._trace = trace
 
+    @classmethod
+    @UREG.wraps(None, (None, _DEFAULT_LEN_UNIT), strict=True)
+    def from_profile_and_length(
+        cls,
+        profile: Union[Profile, VariableProfile, iso.IsoBaseGroove],
+        length: pint.Quantity,
+    ) -> Geometry:
+        """Create a new geometry from a profile and an extrusion length.
+
+        Parameters
+        ----------
+        profile :
+            The cross section profile
+        length :
+            The length of the extrusion
+
+        Returns
+        -------
+        weldx.geometry.Geometry :
+            The new geometry
+
+        """
+        trace = Trace(LinearHorizontalTraceSegment(length))
+        return cls(profile, trace)
```

Comment:
seeing this I had the following idea:
what about we make `__init__` also accept a `pint.Quantity[length]` for trace and in that case it auto generates the `Trace(LinearHorizontalTraceSegment(length))`

no need to remember an extra function name in that case :)